wp-content/themes/borderzine/less/inc/header.less

```diff
@@ -1,4 +1,4 @@
-.dontate-btn {
+.donate-btn {
```

Comment:
Add `.navbar.sticky-navbar .nav-right #header-extras .donate a span`

Reply:
Updated in https://github.com/INN/umbrella-borderzine/pull/56/commits/45eb17855976cb2f1bc17706a334d2b96955a704